invenio_rdm_records/resources/serializers/datacite/schema.py

```diff
@@ -129,7 +135,11 @@ def get_affiliation(self, obj):
                     )
 
                     if id_scheme == "ROR":
-                        identifier_value = "https://ror.org/" + identifier["identifier"]
+                        identifier_value = identifier["identifier"]
```

Comment:
minor: this one important to keep the URL-ified value, because when we register the metadata on DataCite, they prefer to have the full ROR URL (see https://github.com/inveniosoftware/invenio-rdm-records/pull/1156).